### ISSUE TITLE
Bluetooth status and notifications

### ISSUE THREAD

**Issue by avxkim** [trietsch/alfred-bluetooth-connector#1]
BT device status would be useful, when you type `blt`. Now you can only guess if it's connected or disconnected.

**Comment by trietsch** [comment]
I'll have a look whether that's possible with the bluetooth info that is provided by macOS.

**Comment by trietsch** [comment]
Below is the output of what macOS provides when using `PListBuddy`:

```
Array {
    Dict {
        MDR-1000X = Dict {
            device_manufacturer = Cambridge Silicon Radio (0x7, 0x2ABF)
            device_role = attrib_master
            device_supportsEDR = attrib_Yes
            device_services = Headset, Hands-Free unit
            device_isconnected = attrib_Yes
            device_supportsESCO = attrib_Yes
            device_isconfigured = attrib_Yes
            device_majorClassOfDevice_string = Audio
            device_interval = 0 ms
            device_addr = 10-4F-A8-D8-06-ED
            device_ConnectionMode = attrib_active_mode
            device_supportsSSP = attrib_Yes
            device_classOfDevice = 0x04 0x01 0x240404
            device_ispaired = attrib_Yes
            device_RSSI = -46
            device_minorClassOfDevice_string = Headset
        }
    }
}
```

So it should be possible!

**Comment by jsadeli** [comment]
Perhaps could be as simple as this?

Current
```
AirPods
AA-BB-CC-DD-EE-FF
```

After
```
AirPods
AA-BB-CC-DD-EE-FF [Connected]
```

**Comment by trietsch** [comment]
@heihachi88 @jsadeli fixed in [v0.1.1](https://github.com/trietsch/alfred-bluetooth-connector/releases/tag/v0.1.1), also implemented auto-updating, to enable easy future updates.
